src/client/actions/auth.js

```diff
@@ -83,15 +93,24 @@ export function logout(redirect) {
   };
 }
 
-export function signup(firstname, lastname, email, password) {
+export function signup(firstName, lastName, burnName, email, rawDoB, password, repeatpassword) {
```

Comment:
This signature is pretty long. I'd recommend switching to an argument object.

Reply:
Yeah - I thought the same at some point, but it didn't make the final cleanup pass cut. I'll do that and resub.

Reply:
Fixed